### PR TITLE
My first code review

### PR DESCRIPTION
**Pull request by kiplaacollo**
This is a self-code review
it will help me to learn more about how review is done
I will send it for review and review it myself

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Youtube Replica</title>
+<link rel="stylesheet" href="index.css">
+</head>
+<body>
```
Comment (kiplaacollo) [inline index.html]: please Insert a Meta tag so that the header can be compatible with different devices

**Review by kiplaacollo — COMMENTED** [review]
# so close
please update linters on your code

**Review by kiplaacollo — COMMENTED** [review]
# Required Changes 
Please Update linters before you proceed to the next excercise

**Comment by bienvenuushindi** [comment]
> # Required Changes
> 
> Please Update linters before you proceed to the next exercise

[OPTIONAL] I think **Fix** is better than **Update**. How do you see :hugs:

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @kiplaacollo   ,

Nice job :clap: :100: , you are almost there.



## (Highlights) Good Points: :+1:

- :heavy_check_mark: Descriptive PR title & description.
- :heavy_check_mark: Inline comment is clear and easy to understand.

## Required Changes :recycle: 

 - [ ] Your review should have at least two comments. So far you have just one comment. Please add one more comment as requested in the [project requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements)
 > requirement

## [Optional] suggestions:

- [ ] I think it will be great if you use a couple of emojis and/or gifs in the review.They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by kiplaacollo** [comment]
Fix the linters
my project can run smoothly

**Review by kiplaacollo — COMMENTED** [review]
# Great work 👍🏿 
# You can now proceed to the next execise 💯

**Comment by andgarzonmal** [comment]
Hi @kiplaacollo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**